### ISSUE TITLE
Incorrect info on list of Rikai extensions

### ISSUE THREAD

**Issue by Garethp** [sorashi/sorashi.github.io#1]
Hey, just letting you know that RikaiRebuilt works just as well on [Chrome](https://chrome.google.com/webstore/detail/rikairebuilt/bhcfpccmjdpjejaahbblpnikkejnkfcj?p=www.vis.ag) as it does on Firefox. It was actually the main reason I made RikaiRebuilt. Also, the repo can be found [here](https://github.com/Garethp/RikaiRebuilt). I'm also happy to answer any questions you might have about my plugin, since there's no description for it.

**Comment by sorashi** [comment]
Sorry, I did too little digging for information on your extension. I'll fix it asap. Thanks for speaking up.

**Comment by sorashi** [comment]
I tried to describe your extension. See the [article](https://sorashi.github.io/comprehensive-list-of-rikai-extensions/) and tell me if you are not satisfied.

Again, I'm sorry -- I didn't think people would actually read the article. I wrote it for myself and my future self. But again, it's quite high in Google searches for "rikai extensions" and "rikai list". So I guess my responsibility is higher now.
